modules/items/server/controllers/items.server.controller.js

```diff
@@ -38,7 +38,9 @@ exports.createCategory = function (req, res) {
 };
 
 exports.deleteCategory = function(req, res) {
-  var toDelete = req.body;
+  console.log(req);
```

Comment:
morgan will auto log stuff for us, we dont need these console statements

Reply:
Good catch.